Aptoide/aptoide-client-v8
# [DT-3] Get ads conversion

**What does this PR do?**

   Create and use a campaigns module. 

   Caching companies in memory to avoid duplicated network calls.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] CampaignsUseCaseTest.kt
- [ ] CampaignImplTest.kt

**How should this be manually tested?**

  Run unit tests.
  Run and check the requests in the logs:
  1. Open the aptoide
  2. Open the app details with campaign
  3. Check the network calls

**What are the relevant tickets?**

  Tickets related to this pull-request: [DT-3](https://aptoide.atlassian.net/browse/DT-3)



**Code Review Checklist**
